alb/variables.tf

```diff
@@ -78,3 +78,12 @@ variable "target_id" {
 variable "vpc_id" {
   description = "VPC ID"
 }
+
+variable "own_host_header_only" {
+  default = false
```

Comment:
Maybe a description for the variable would be helpful?

Reply:
Yep, good point. I've added one in.